lib/runner/testcase.js

```diff
@@ -43,6 +55,8 @@ TestCase.prototype.run = function () {
   this.results = null;
   this.errors = null;
 
+  this.suite.client.setMaxListeners(this.countTests()+1);
```

Comment:
Yep, i know. But don't know consequences of this solution. So made it exactly such as required. Btw in nightwatch itself it's not set event to "0" or other value. And this is rather annoying in every big test suite.